### PR TITLE
add Landkreis Nienburg/Weser

### PR DESCRIPTION
**Pull request by Juphex**
request url taken from
https://www.lk-nienburg.de/portal/seiten/corona-startseite-901000979-21500.html
i.e.:
https://gis-nienburg.maps.arcgis.com/apps/opsdashboard/index.html#/059f481c5cc24319b8a983cd90cc2a94

### CODE REVIEW

File: landkreise/get-nienburg-kreis.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+json_data = json.loads(text)
+
+cases = json_data["features"][0]["attributes"]["Faelle_bestaetigt"]
+unix_timestamp = int(json_data["features"][0]["attributes"]["Tagesdatum"])
+unix_timestamp /= 1000
+
+#convert unix to date
+status = datetime.datetime.fromtimestamp(unix_timestamp).strftime("%Y-%m-%d")
+
+add_to_database("03256022", status, cases, "Landkreis Nienburg/Weser")
```
Comment (dadosch) [inline landkreise/get-nienburg-kreis.py:29]: This is about the landkreis, not the city; right? Then the AGS is actually 03256 . The AGS you wrote is from the city Nienburg/Weser.

Comment (Juphex) [reply landkreise/get-nienburg-kreis.py:29]: Thanks. Source?

Comment (dadosch) [reply landkreise/get-nienburg-kreis.py]: For example here on wikipedia: https://de.wikipedia.org/wiki/Landkreis_Nienburg/Weser

or in the csv/json linked in the readme (note the different strings in the beginning of each line there).

Out of curiosity: How did you find the project?

Comment (Juphex) [reply landkreise/get-nienburg-kreis.py]: Tracking the requested websites/data api's when loading the website gave me a clue.

**Pull request merged**